### PR TITLE
Translate Failsafe Stage 1 options, improve explanation of function "Time in stage 1"

### PR DESCRIPTION
**Pull request by HThuren**
Translate Auto, Hold and Set for Stage 1

failsafeDelay and failsafeDelayHelp corrected to:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:4581]: This should be stage 1.

Comment (haslinghuis) [reply locales/en/messages.json:4581]: Sure @ctzsnooze would have noticed

Comment (HThuren) [reply locales/en/messages.json:4581]: Waiting for comment from @ctzsnooze or just rollback ? :-)

Comment (ctzsnooze) [reply locales/en/messages.json:4581]: Good catch, the wording is unclear.
What actually happens is that a signal loss more than than 30ms is starts a counter, and after 100ms of no signal we initiate Stage 1 behaviour.  We stay in Stage 1 until the failsafe delay time expires, and then enter Stage 2.  While in Stage 1, signal recovery causes immediate exit from Stage 1 behaviour.

Perhaps we can say, "The period of time in Stage 1 failsafe after signal loss.  If the delay time expires without return of signal, Stage 2 failsafe will start."

Does that sound OK?

Comment (HThuren) [reply locales/en/messages.json:4581]: Good suggestion, now:
image

**Comment by HThuren** [comment]
> @ctzsnooze, by the way, rename this image to Stage 1 - Channel Fallback Settings ?

@ctzsnooze, have you noticed this, if we on Failsafe do change "Channel Fallback Settings" to "Stage 1 - Channel Fallback Settings" ?

**Comment by HThuren** [comment]
@ctzsnooze @haslinghuis can you approve this PR ?

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
